-- @Exception

function foo(x)
  echo x
end

echo foo(2, 3)
